services/drupal/web/modules/custom/epa_core/epa_core.module

```diff
@@ -373,13 +374,45 @@ function epa_core_tokens($type, $tokens, array $data, array $options, Bubbleable
           $term_description = $token->replace($term_description, ['term' => $data['term']]);
           $replacements[$original] = $term_description;
         }
-
         break;
     }
   }
   return $replacements;
 }
 
+function epa_core_addtocal_alter(array &$overridable_items) {
+  $overridable_items['override_status'] = TRUE;
+  $node = \Drupal::routeMatch()->getParameter('node');
+  if ($node->bundle() == 'event') {
+    $cal_description = $node->get('field_add_to_cal_description')->value;
+    $default_description = $node->get('field_description')->value;
+    if (!empty($cal_description)) {
+      $description = "$cal_description" . "\n";
+    } elseif (!empty($default_description)) {
+      $description = "$default_description" . "\n";
```

Comment:
Same issue as above with funky concatenation and double-quotes, but let's just drop the concatenation and \n entirely as we'll add the line breaks at the end.